src/app/transfer-funds/transfer-funds.component.html

```diff
@@ -157,6 +157,39 @@ <h3 class="b-rh-1 b-bold">Transfer Donation Funds</h3>
                         <button mat-button matStepperNext>Next</button>
                     </div>
                 </mat-step>
+              <mat-step
+                formGroupName="marketing"
+                [stepControl]="marketingGroup"
+              >
+                <ng-template matStepLabel>Receive Updates</ng-template>
+
+
+                <p class="binary-question-header b-bold b-rt-0" id="optInTbgEmail-label">Would you be happy to receive emails from the <span class="colour-primary">Big Give</span>?</p>
```

Comment:
We should probably delete the word `the`, both here and on the donation start form where I copied this from.

Reply:
I agree – happy for it to be in this PR or another.

We might also want to think about removing the emphasis span around the org name here, as this form has only one comms question?

Reply:
Done in e4b0e336